engine/src/logger/instance.cpp

```diff
@@ -12,277 +12,323 @@
 	#include <Windows.h>
```

Comment:
Prefer the "system/win32/windowsHeader.hpp" include in genesis over  the "Windows.h" include as it is far more light weight and removes the vast majority of unnecessary bloat brought in by the winapi.